### PR TITLE
[filter] Add custom props for snpe filter to support various runtimes

### PR DESCRIPTION
**Pull request by anyj0527**
- Let `custom=Runtime:[CPU|GPU|DSP|NPU]` set each snpe runtime.

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

File: ext/nnstreamer/tensor_filter/tensor_filter_snpe.cc [hunk 1]
```diff
@@ -277,14 +313,20 @@ snpe_subplugin::configure_instance (const GstTensorFilterProperties *prop)
...
+
+    /* Check the given Runtime is available */
+    std::string runtime_str = std::string (Snpe_RuntimeList_RuntimeToString (runtime));
     if (Snpe_Util_IsRuntimeAvailable (runtime) == 0)
-      throw std::runtime_error ("Runtime " + runtime_str + " is not available");
+      throw std::runtime_error ("Given runtime " + runtime_str + " is not available");
+
+    nns_logi ("Given runtime %s is available", runtime_str.c_str ());
 
+    /* set runtimlist config */
```
Comment (niley7464) [inline ext/nnstreamer/tensor_filter/tensor_filter_snpe.cc:326]:
```suggestion
    /* set runtimelist config */
```

**Review by niley7464 — APPROVED** [review]
LGTM except for a minor typo

File: ext/nnstreamer/tensor_filter/tensor_filter_snpe.cc (hunk 1 above)
Comment (anyj0527) [reply ext/nnstreamer/tensor_filter/tensor_filter_snpe.cc]: I fixed it as your suggestion. Thanks!

**Review by myungjoo — APPROVED** [review]

**Review by wooksong — APPROVED** [review]

**Comment by wooksong** [comment]
It is possible to merge after resolving the conflicts.

**Comment by anyj0527** [comment]
Oops this change is merged in https://github.com/nnstreamer/nnstreamer/pull/4487. Close it.

**Pull request closed without merging**